tests/foundry/exclude

```diff
@@ -85,6 +85,7 @@ SafeTest.testWithdrawFuzz
 SetUpTest.testSetUpCalled
 SetUpTest.testSetUpCalledSymbolic
 SignTest.testSign
+SymbolicStorageTest.testSymbolicStorage
```

Comment:
```suggestion
SymbolicStorage.testSymbolicStorage
```

Reply:
changed the contract name instead (to be consistent with the other tests).